app/web/src/components/IconButton.tsx

```diff
@@ -19,14 +19,14 @@ interface IconButtonProps extends MuiIconButtonProps {
   loading?: boolean;
 }
 
-export default function IconButton({
-  loading,
-  ...otherProps
-}: IconButtonProps) {
+export default forwardRef(function IconButton(
+  { loading, ...otherProps }: IconButtonProps,
+  ref: Ref<any>
```

Comment:
Where is this ref being used?

Also, could probably do this for better types:
```suggestion
  ref: Ref<IconButtonProps["ref"]>
```

Reply:
I was getting warnings about unforwarded refs. We will also have to do this for a lot of components if we upgrade to mui v5.